src/navigation/convert_GPS_waypoints.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+import numpy as np
+import rospy
+
+from mrover.msg import EnableAuton,  Waypoint, GPSWaypoint
+from util.course_service import CourseService
+
+from util.SE3 import SE3
+from util.tf_utils import EARTH_RADIUS
+
+class Converter:
+    def __init__(self):
+        rospy.init_node("gps-converter")
+        rospy.Subscriber("auton/enable_state", EnableAuton, self.read_data)
+        self.publish_course = CourseService()
+    
+    
+    def convert(self, waypoint: GPSWaypoint):
+        """
+        Converts a GPSWaypoint into a "Waypoint" used for publishing to the CourseService.
+
+        GPSWaypoint:
+        float latitude_degrees
+        float longitude_degrees
+        bool gate #should we search for a gate around the above coordinate
+        bool post #should we search for a post around the above coordinate (NOT RELEVANT IF GATE IS TRUE)
+        int32 id
+
+        Waypoint:
+        int32 fiducial_id
+        string tf_id
+        """
+
+        # Create odom position based on GPS latitude and longitude
+        base = np.array([42.2, -83.7, 0.0]) # Our base/reference is in latitude, longitude, altitude coordinates (degrees)
+        x = EARTH_RADIUS * np.radians(waypoint.latitude_degrees - base[0])
+        y = EARTH_RADIUS * np.radians(waypoint.longitude_degrees - base[1])
```

Comment:
@rbridges12 thought we wanted to use a more accurate elliptical model for this. E.g. wgs84?

Reply:
will update this with the [pymap3d library](https://geospace-code.github.io/pymap3d/enu.html) once it is confirmed to work for localization

Reply:
pymap3d is now being used!